services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/ovs/DataPlaneServiceImpl.java

```diff
@@ -266,10 +252,8 @@ private void buildPortState(NetworkConfiguration networkConfig, List<InternalPor
 
     private NeighborState buildNeighborState(NeighborEntry neighborEntry, NeighborInfo neighborInfo, OperationType operationType) {
         NeighborConfiguration.Builder neighborConfigBuilder = NeighborConfiguration.newBuilder();
-        neighborConfigBuilder.setFormatVersion(FORMAT_REVISION_NUMBER);
         neighborConfigBuilder.setRevisionNumber(FORMAT_REVISION_NUMBER);
-        //neighborConfigBuilder.setId();
-        //neighborConfigBuilder.setProjectId();
+        //neighborConfigBuilder.setId(); // !!!We are going to need this per latest ACA change!!!
```

Comment:
Instead of "!!!", let us use //TODO to remind us for real.

Reply:
@kevin-zhonghao can help to add this in next DPM change.

Reply:
Fixed. I like to use !!! to get people's attention. :)